src/app/containers/RadioPageMain/__snapshots__/index.test.jsx.snap

```diff
@@ -602,6 +603,7 @@ exports[`Radio Page Main should match snapshot for Canonical 1`] = `
 }
 
 <html
+  classname="js"
```

Comment:
something is not quite right here. I though react testing library snapshots would output class 🤔

Reply:
Yeh search for `classname="` in all .snap files and you won't find any other use of it. It should output `class="` like https://github.com/bbc/simorgh/blob/latest/src/app/containers/FrontPageSection/__snapshots__/index.test.jsx.snap#L525-L540